src/ussd_higherhealth_healthcheck.js

```diff
@@ -121,14 +120,22 @@ go.app = (function () {
           "",
           "Reply"
         ].join("\n"));
-      } else {
-        question = $([
-          "HIGHER HEALTH HealthCheck is your risk assessment tool.",
-          "No result SMS will be sent. Continue or WhatsApp HI to 0600110000",
-          "",
-          "Reply"
-        ].join("\n"));
+        return new MenuState(name, {
+          question: question,
+          error: $("This service works best when you select numbers from the list"),
+          accept_labels: true,
+          choices: [
+            new Choice("state_terms", $("START")),
+            new Choice("state_display_risk", $("RECEIPT"))
```

Comment:
in `state_start`, all the answers are replaced with what is found in the user's profile, if they have one. That means that we won't have the information required to be able to display a receipt. 

Also, the `answers` data is only kept around for an hour, so if we need to be able to show a receipt for longer than that, we can't rely on the answers data.

We probably need to add to the current healthcheck API to allow fetching the healthchecks (or last healthcheck) for a user (making sure that it's indexed well and performant), and then use that API to fetch their last healthcheck, making sure that it's within some timeframe (eg. you should only be able to view your receipt for 24 hours, or more likely, you can only view the receipt on the same day that you did the healthcheck). We would also probably not want to show the `RECEIPT` option if they haven't completed a healthcheck within the given timeframe.